crates/css-syntax/src/syntax.rs

```diff
@@ -631,6 +555,14 @@ pub enum SyntaxInput<'a> {
     Css(CssType<'a>),
 }
 
+fn scope_from_browser_compat(browser_compat: Option<&str>) -> Option<&str> {
+    if let Some(bc) = browser_compat {
```

Comment:
Nit: Maybe exclude non-CSS keys explicitly, to avoid surprises.

```suggestion
    if let Some(bc) = browser_compat && bc.starts_with("css.") {
```